src/js/lib/content/firebase.yaml

```diff
@@ -0,0 +1,8 @@
+apiKey: "AIzaSyBDOiRrSAEW_qGqBPzaCmBeSl2e9L7jt3s"
+authDomain: "enso-ide.firebaseapp.com"
+projectId: "enso-ide"
+storageBucket: "enso-ide.appspot.com"
+messagingSenderId: "724044315337"
+appId: "1:724044315337:web:7e4530f3663ba7a3e26b18"
+measurementId: "G-FJXM0PWBYR"
+clientId: "724044315337-itp6glm1v6grinmus68qp267lpublpvu.apps.googleusercontent.com"
```

Comment:
all these thigns can be public , right?

Reply:
I regenerated keys and created a GH secret.